tokio/src/macros/select.rs

```diff
@@ -608,6 +608,10 @@ macro_rules! select {
 
     // ===== Entry point =====
 
+    ($(biased;)? else => $else:expr $(,)? ) => {
+        $else
```

Comment:
You need to wrap them in another pair of `{}` braces. That reduces the risk of it expanding to something wrong.